websites/H/HBO Max/presence.ts

```diff
@@ -65,30 +67,67 @@ function getToken(): Promise<string> {
 					paymentProviderCode: "blackmarket"
 				}
 			})
+			/* eslint-enable camelcase */
 		})
-			.then(x => x.json())
-			.then(x => resolve(x.access_token));
-	});
+	).json();
+
+	return response.access_token;
 }
-/* eslint-enable camelcase */
 
-async function fetchCover() {
+async function fetchClientConfig(token: string) {
 	const response = await (
-		await fetch(
-			`https://comet.api.hbo.com/express-content/${
-				location.pathname.split("/")[2]
-			}?device-code=desktop&product-code=hboMax&api-version=v9.0&country-code=US&language=en-us`,
-			{
-				headers: {
-					authorization: `Bearer ${await getToken()}`
-				}
-			}
-		)
+		await fetch("https://sessions.api.hbo.com/sessions/v1/clientConfig", {
+			method: "POST",
+			headers: {
+				authorization: `Bearer ${token}`,
+				"content-type": "application/json"
+			},
+			body: JSON.stringify({
+				contract: "abc:1.0.0.0",
+				preferredLanguages: ["en-us"]
+			})
+		})
 	).json();
 
-	return `https://artist.api.cdn.hbo.com/images/${
-		response[0].body.references.series.match(/series:([^:]+)/)[1]
-	}/tileburnedin?size=1024x1024`;
+	return [
+		response.routeKeys.contentSubdomain,
+		new URLSearchParams(
+			response.features["express-content"].config.expressContentParams
+		).get("country-code")
+	];
+}
+
+async function fetchCover() {
+	let output: string;
+
+	const accessToken = await fetchToken(),
+		[routeKey, countryCode] = await fetchClientConfig(accessToken);
```

Comment:
Use Promise.all

Reply:
![image](https://user-images.githubusercontent.com/54318514/173210278-020eb542-8df1-4fa6-9a53-5375161ea099.png)
probably would be better to keep it this way

Reply:
Oh of course, my bad